src/xdocs/config_coding.xml

```diff
@@ -3588,7 +3588,31 @@ for (String line: lines) {
         <source>
 &lt;module name=&quot;NestedTryDepth&quot;/&gt;
         </source>
-
+        <p>
+            Example of violation:
+        </p>
+        <source>
+try {
+    try {
+        try { // violation: Max default allowed depth is 1
```

Comment:
please expand message so it is clear what the current depth is along with the max allowed depth. This helps connect how we are counting so users are clear.
Example: `violation: Current depth is X, max depth allowed is Y`.